crates/sui-bridge/src/crypto.rs

```diff
@@ -221,4 +228,96 @@ mod tests {
+
+        let public_key_bytes =
+            Hex::decode("02321ede33d2c2d7a8a152f275a1484edef2098f034121a602cb7d767d38680aa4")
+                .unwrap();
+        let pubkey1 = BridgeAuthorityPublicKey::from_bytes(&public_key_bytes).unwrap();
+        let authority1 = BridgeAuthority {
+            pubkey: pubkey1.clone(),
+            voting_power: 2500,
+            is_blocklisted: false,
+            base_url: "".into(),
+        };
+
+        let public_key_bytes =
+            Hex::decode("027f1178ff417fc9f5b8290bd8876f0a157a505a6c52db100a8492203ddd1d4279")
+                .unwrap();
+        let pubkey2 = BridgeAuthorityPublicKey::from_bytes(&public_key_bytes).unwrap();
+        let authority2 = BridgeAuthority {
+            pubkey: pubkey2.clone(),
+            voting_power: 2500,
+            is_blocklisted: false,
+            base_url: "".into(),
+        };
+
+        let public_key_bytes =
+            Hex::decode("026f311bcd1c2664c14277c7a80e4857c690626597064f89edc33b8f67b99c6bc0")
+                .unwrap();
+        let pubkey3 = BridgeAuthorityPublicKey::from_bytes(&public_key_bytes).unwrap();
+        let authority3 = BridgeAuthority {
+            pubkey: pubkey3.clone(),
+            voting_power: 2500,
+            is_blocklisted: false,
+            base_url: "".into(),
+        };
+
+        let public_key_bytes =
+            Hex::decode("03a57b85771aedeb6d31c808be9a6e73194e4b70e679608f2bca68bcc684773736")
+                .unwrap();
+        let pubkey4 = BridgeAuthorityPublicKey::from_bytes(&public_key_bytes).unwrap();
+        let authority4 = BridgeAuthority {
+            pubkey: pubkey4.clone(),
+            voting_power: 2500,
+            is_blocklisted: false,
+            base_url: "".into(),
+        };
+
+        let committee = BridgeCommittee::new(vec![
+            authority1.clone(),
+            authority2.clone(),
+            authority3.clone(),
+            authority4.clone(),
+        ])
+        .unwrap();
+
+        let action = BridgeAction::SuiToEthBridgeAction(SuiToEthBridgeAction {
+            sui_tx_digest: TransactionDigest::random(),
+            sui_tx_event_index: 0,
+            sui_bridge_event: EmittedSuiToEthTokenBridgeV1 {
+                nonce: 1,
+                sui_chain_id: BridgeChainId::SuiTestnet,
+                sui_address: SuiAddress::from_str(
+                    "0x80ab1ee086210a3a37355300ca24672e81062fcdb5ced6618dab203f6a3b291c",
+                )
+                .unwrap(),
+                eth_chain_id: BridgeChainId::EthSepolia,
+                eth_address: EthAddress::from_str("0xb18f79Fe671db47393315fFDB377Da4Ea1B7AF96")
+                    .unwrap(),
+                token_id: TokenId::ETH,
+                amount: 100000u64,
+            },
+        });
+        let sig = BridgeAuthoritySignInfo {
+            authority_pub_key: pubkey1,
+            signature: BridgeAuthorityRecoverableSignature::from_bytes(
```

Comment:
How to update this value in future if action format changes?

Reply:
I have a cli tool locally (haven't checked in) locally to generate this. As we add more e2e test cases, we can use them to generate sigs as well

Reply:
A comment / doc linking to the tool will be helpful in future, in case someone else needs to update the format / test.